app/src/main/res/layout/browser_tabstray_item.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="72dp">
+
+    <ImageView
+        android:id="@+id/mozac_browser_tabstray_icon"
+        android:layout_width="100dp"
+        android:layout_height="56dp"
+        android:visibility="gone"
+        android:importantForAccessibility="no"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.card.MaterialCardView
```

Comment:
I noticed that there was a clipping on thumbnails in order to prevent [drawing outside the corners](https://github.com/material-components/material-components-android/issues/911#issuecomment-579434787). I looked into disabling that, but it makes the tabs look subpar.

Let's step away from `MaterialCardView` in that case (unfortunately) and go back to using a regular `CardView`. We can then add some `marginStart` to 8dp to align it better. This doesn't match the designs, but it's the best we can do with this limitation.